CIP-0037/CIP-0037.md

```diff
@@ -0,0 +1,217 @@
+###### tags: `CIP`
```

Comment:
Suggestions: 

Also expose non-pairing curves using same API (except for pairing).